src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Actions/Row/edit_customer_discount.html.twig

```diff
@@ -36,6 +36,8 @@
     data-placement="top"
     data-original-title="{{ action.name }}"
   {% endif %}
+   data-confirm-message
```

Comment:
But where is the confirm message ?

Reply:
@Progi1984 There is no need to have a confirm message to edit a customer but the attribute is required in order to make the JS extension working

Reply:
```suggestion
   data-confirm-message=""
```

In this case, that allow us to understand better.